### DISCUSSION TITLE
Funds not arrived at Solana add - pending for days at "Awaiting completion on Solana"

### DISCUSSION THREAD

**Discussion by chandanverma8** [wormhole-foundation/wormhole#209] [Problems / bugs] [answered]
Hi,

I transferred USDC to Solana address from Eth address through https://s4b43ur3ynpgozahlqhnxmsye6j2ei27wc2coke5unbgmcpoy5rq.arweave.net/lwPN0jvDXmdkB1wO27JYJ5OiI1-wtCconaNCZgnux2M/#/

Transaction has completed in ETH, however it is still pending at "Awaiting completion on Solana" and funds not received at Solana address. Transaction hash link is as below.

https://etherscan.io/tx/0xdac01d8b67f06e2eccca6a3737b5fb5b27c0327d463483282996ee632c36f293

When i did subsequent transaction for fewer cents, it went through and i have received the funds. Below is link of completed transaction hash.

https://etherscan.io/tx/0xc94270709e1430a94f1d159f77ae8e84b2020eb0c53f4f6dcf20b4a8282b1659

My solana address is as below.

3Ax3xYC9JTUjmDWjPHjx5JR6WWtuuokCvjHD1P97cUks

Please resolve.

Regards,
Chandan Verma

**Comment by hendrikhofstadt** [comment, marked as answer]
Your message got stuck, we're sorry for the inconveniences caused by this.
I just relayed the message again and your coins have arrived.

**Reply by chandanverma8** [reply]
Thank you so much for the support !!!
